docs/features/basic-concepts/genesis-file.md

```diff
@@ -4,13 +4,15 @@ The Genesis file is the basis for the entire network initialization，which cont
 
 ## Basic State
 
-* genesis_time The time to build Genesis file
-* chain_id     Blockchain’s ID
+* **genesis_time** The time to build Genesis file
+* **chain_id**     Blockchain’s ID
 
 ## Consensus Params
 
-* block_size Block size and config params of the number of Gas in the block
-* evidence   The lifecycle of deception evidence in the block
+* **block_size** 
+  * `max_bytes` The max size of a block
+  * `max_gas`  The max Gas number of a block, default value is -1 and means no gas limit. If the block gas limit exceed, the transaction and subsequent transactions will fail to deliver. 
```

Comment:
`max_gas`:  The maximum Gas quantity a block. Its default value is `-1` which means no gas limit. If the accumulation of comsumed gas exceeds the block gas limit, then all subsequent transactions in the same block will fail to deliver.